<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEditorClass</class>
 <widget class="QMainWindow" name="FileEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>431</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Text</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="gfxPreview">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_FILE_Editor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolTip">
    <string>Move forward</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="actionUp"/>
   <addaction name="actionDown"/>
   <addaction name="actionGo_back"/>
   <addaction name="actionGo_forward"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_FILE_Editor"/>
  </widget>
  <action name="actionAbout_FILE_Editor">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/About.png</normaloff>:/FileEditor/Resources/About.png</iconset>
   </property>
   <property name="text">
    <string>About FILE Editor</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Load.png</normaloff>:/FileEditor/Resources/Load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Save.png</normaloff>:/FileEditor/Resources/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Exit.png</normaloff>:/FileEditor/Resources/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGo_back">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Back.png</normaloff>:/FileEditor/Resources/Back.png</iconset>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </action>
  <action name="actionGo_forward">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Forward.png</normaloff>:/FileEditor/Resources/Forward.png</iconset>
   </property>
   <property name="text">
    <string>Go forward</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Thumbs up.png</normaloff>:/FileEditor/Resources/Thumbs up.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Thumbs down.png</normaloff>:/FileEditor/Resources/Thumbs down.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Up.png</normaloff>:/FileEditor/Resources/Up.png</iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
   <property name="toolTip">
    <string>Move before</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="icon">
    <iconset resource="FileEditor.qrc">
     <normaloff>:/FileEditor/Resources/Down.png</normaloff>:/FileEditor/Resources/Down.png</iconset>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
   <property name="toolTip">
    <string>Move after</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileEditor.qrc"/>
 </resources>
 <connections/>
</ui>
